src/util/sample.cpp

```diff
@@ -480,6 +480,14 @@ void SampleUtil::copyClampBuffer(CSAMPLE* M_RESTRICT pDest,
     }
 }
 
+// static
+void SampleUtil::applyClamp(CSAMPLE* pBuffer, SINT iNumSamples) {
+    // note: LOOP VECTORIZED.
```

Comment:
did you verify that this loop is vectored? 
https://github.com/mixxxdj/mixxx/blob/59ca3a25b768c44ac0934ed4cb4504cac52bde54/src/util/sample.cpp#L16

Reply:
No, I have no clue what it is. It's just copy-paste of the original method. I'll remove this note.

Reply:
You might want to make sure it is  vectorized instead as it is [a significant optimisation](https://github.com/mixxxdj/mixxx/blob/ba1ac4545e821f3ab8bea602f002e416a26b8397/src/util/sample.h#L56). You can do so by using the `INFO_VECTORIZE` CMake option. In the build output, you should see an info/notice with the file and the line (`src/util/sample.cpp:483`) saying something along the lines of "vectorized with block of x bytes`